### ISSUE TITLE
Change the publish frequency

### ISSUE THREAD

**Issue by hoschiking** [nygma2004/growatt2mqtt#41]
I would like to change the publish frequency from 4 to ~30 seconds.
Which code I have to change for this? where is the frequency defined?
thank you in advance

**Comment by nygma2004** [comment]
Try in settings.h and change the MODBUS_UPDATE  and change it to 15.

**Comment by hoschiking** [comment]
Thank you, it works :-)
one more question
usually I sould be able to change the frequency and another paramters under the topc /growatt/write
the problem is, the folder /write does not exist. do you have any idea why?
I switsched to expert level in ioBroker and I´m not able to creat this folder.
![2024-08-01 12_28_50-Window](https://github.com/user-attachments/assets/cb476e68-651f-4b0b-8f99-27d796dda680)
I´m missing all writable parameters.
![2024-08-01 12_30_20-Window](https://github.com/user-attachments/assets/a7d13441-507c-4a86-859d-23bad25cb140)

**Comment by nygma2004** [comment]
Yes, this project only implements these few parameters to be writable, rest is fixed in the code.
